internal/envstest/bootstrap.go

```diff
@@ -67,11 +72,27 @@ func Bootstrap(db *database.Database) (*BootstrapResponse, error) {
 
 	realm.AllowedTestTypes = database.TestTypeNegative | database.TestTypeConfirmed | database.TestTypeLikely
 	realm.AllowBulkUpload = true
+	realm.UseAuthenticatedSMS = true
 	if err := db.SaveRealm(realm, database.SystemTest); err != nil {
 		return &resp, fmt.Errorf("failed to save realm: %w", err)
 	}
 	resp.Realm = realm
 
+	// Ensure an SMS signing key exists.
+	if _, err := realm.CurrentSMSSigningKey(db); err != nil {
```

Comment:
does the e2e runner have key admin capabilities?

Reply:
Good call. We could alternatively remove this and move it into the admin setup guide with the existing manual steps. What do you think? I'm kinda worried about giving e2e permission to manage keys.

Reply:
my vote is move it to the docs

Reply:
Done